awscli/customizations/configure.py

```diff
@@ -302,6 +303,9 @@ def _run_main(self, args, parsed_globals):
         region = self._lookup_config('region')
         self._display_config_value(region, 'region')
 
+        ca_bundle = self._lookup_config('ca_bundle')
```

Comment:
I understand your point, but as it is, there is no way to verify where the `ca_bundle` option is getting its value. That kind of seems even less ideal to me. But oh well, I suppose I'll just open a new issue for the `aws configure list --all` CLI option to keep it on the radar.